models/userCourses.js

```diff
@@ -0,0 +1,53 @@
+module.exports = {
+    name: 'userCourses',
+    schema: {
+      status: {
```

Comment:
@praveenKDass userCourse require all these fields? all these are mentioned in the doc a well